lib/basedriver/driver.js

```diff
@@ -412,7 +413,7 @@ class BaseDriver extends Protocol {
 
     // Construct the arguments for createSession depending on the protocol type
     const args = this.protocol === PROTOCOLS.W3C ?
-      [undefined, undefined, {alwaysMatch: this.caps, firstMatch: [{}]}] :
+      [undefined, undefined, this.originalCaps] :
```

Comment:
should probably add a TODO to do away with sending a whole array once we no longer have the concept of JSONWP caps?